src/requirements.txt

```diff
@@ -1,7 +1,7 @@
 Flask==2.1.3
 urllib3==1.26.5
 jsonschema==3.2.0
-
+hubmap-sdk==1.0.2
```

Comment:
@DerekFurstPitt you'll also need to remove this dependency otherwise the build still fails.